src/openforms/appointments/contrib/qmatic/tests/data/vcr_cassettes/ListAvailableProductsTests/ListAvailableProductsTests.test_listing_products_with_location_and_product_constraint.yaml

```diff
@@ -0,0 +1,440 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiIiLCJpYXQiOjE2OTM5MjU5OTgsImNsaWVudF9pZCI6IiIsInVzZXJfaWQiOiIiLCJ1c2VyX3JlcHJlc2VudGF0aW9uIjoiIn0.mqpHvBBHEsCM_8A9CnBnDLrcgqeq6Ees-lNVXnvdFRU
```

Comment:
Are these disposable tokens? Or should they have been [filtered](https://vcrpy.readthedocs.io/en/latest/advanced.html#filter-sensitive-data-from-the-request) from the cassette?

Reply:
They are a mistake in the service factory usage, it should be set up with no auth instead of zgw auth. I reckon it's now producing a token for client id `""` and secret `""`. I checked them on jwt.io and realized that they're not an issue.

I'd prefer making a separate PR to address this so this can get merged, is that okay?